src/modules/partition/jobs/CreatePartitionJob.cpp

```diff
@@ -46,6 +172,21 @@ CreatePartitionJob::prettyName() const
 QString
 CreatePartitionJob::prettyDescription() const
 {
+    if ( CalamaresUtils::Partition::getPartitionTable( m_partition )->type() == PartitionTable::TableType::gpt )
+    {
+        QString entries = prettyGptEntries( m_partition );
+        if ( entries.size() )
```

Comment:
More idiomatic would be `!entries.isEmpty()`, since it's a string. The same comments about translatability apply here.

Reply:
Make sense.